.mailmap

```diff
@@ -0,0 +1,4 @@
+Brandon Philips <brandon.philips@coreos.com> <brandon@ifup.org>
+Doug Davis <dug@us.ibm.com> <duglin@users.noreply.github.com>
+Sergiusz Urbaniak <sergiusz.urbaniak@gmail.com> <sur@coreos.com>
+xiekeyang <keyang.xie@gmail.com> <xiekeyang@huawei.com>
```

Comment:
From your [PR comment][1], @xiekeyang uses the Huawei address more frequently than the Google address.  He might prefer you swap the order of addresses here to make the Huawei address canonical.

Similarly with @s-urbaniak in the line above.  You have the CoreOS address as more common, but the line above is making the Google address canonical.  And again for the @philips line.

Anyhow, the email these folks consider canonical is up to them, and we can always adjust the canonical choice if we hear from them later, but I'd have guessed a few of them the other way around ;).
https://github.com/opencontainers/image-tools/pull/186#issue-260471725

Reply:
updated.